crypto/keys/multisig/multisig.go

```diff
@@ -148,6 +148,16 @@ func (m *LegacyAminoPubKey) Type() string {
 	return "PubKeyMultisigThreshold"
 }
 
+// MarshalAmino overrides amino binary marshaling.
+func (m LegacyAminoPubKey) MarshalAmino() ([]byte, error) {
```

Comment:
```suggestion
func (m *LegacyAminoPubKey) MarshalAmino() ([]byte, error) {
```

Reply:
We can't use a pointer there, otherwise it won't conform to the interface correctly